README.md

```diff
@@ -118,18 +118,18 @@ The `listen` property of the YAML config file defines how the demo server's
 network listener is configured. This property is a YAML map with the following
 keys:
 
-* `bindAddress`: The address on which to listen. This defaults to 127.0.0.1, so
+* **`bindAddress`**: The address on which to listen. This defaults to 127.0.0.1, so
```

Comment:
Do you think it is better if these reflect their yaml names, like `bind_address`?

Reply:
PR to fix case: #2

Reply:
Yeah, looks like several things I missed in the README back when I renamed all of the yaml keys (they used to be camel-case...).